src/js/components/Charts/Pie/index.js

```diff
@@ -0,0 +1,157 @@
+import React, { Component } from "react"
```

Comment:
since this file doesn't have dependencies, you can just move it to `/Charts/Pie.js`